docs/ports/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Input and Output Ports
+excerpt: "Documentation reference for the ports that one plugs sensors and motors into on the EV3 and their associated ev3dev drivers."
```

Comment:
This wording is a bit awkward to me. How about "Documentation reference for the input (sensor) and output (motor) port device drivers."